src/backend/index.ts

```diff
@@ -11,17 +14,38 @@ import descriptionBulletsRouter from './src/routes/description-bullets.routes';
 const app = express();
 const port = process.env.PORT || 3001;
 
+const allowedHeaders =
+  process.env.NODE_ENV === 'production'
+    ? 'Origin, X-Requested-With, Content-Type, Accept, Authorization, XMLHttpRequest'
+    : '*';
+
 const options: cors.CorsOptions = {
-  origin: '*',
+  origin: ['http://localhost:3000', 'https://finishlinebyner.com', 'https://magenta-mochi-275e56.netlify.app'],
```

Comment:
is there a QA frontend url to add? I assume that's the last one, but adding a sub domain for it might be nice.

Reply:
A common approach is api.host.com, qa.api.host.com, for API and QA API respective. host.com and qa.host.com for the frontend.

Reply:
good idea